src/app/_services/api.service.ts

```diff
@@ -173,6 +172,10 @@ export class ApiService {
     return this.httpClient.get<HousingAvailable[]>(`${this.apiUrl}/housing-available`);
   }
 
+  addHousingAvailable(housingAvailable: HousingAvailable) {
+    return this.httpClient.post<HousingAvailable>(`${this.apiUrl}/housing-available`, housingAvailable, {observe: 'response'});
```

Comment:
{observe: 'response'} should only be needed when we need to drop into the particulars of the request response (used for account's posting because depending on the response code we do different things, and 'ok' or 'bad' isn't good enough in that case. I don't think we're doing anything like that here.